internal/utils/csi.go

```diff
@@ -390,3 +430,9 @@ func FuseMountOptionsContainerArg(options map[string]string) string {
 		"",
 	)
 }
+
+func Privileged() *corev1.SecurityContext {
+	return &corev1.SecurityContext{
+		Privileged: ptr.To(true),
+	}
+}
```

Comment:
Looking at the call sites, it does not make any sense to have a util that creates the privileged security context but not the unprivileged security context. Let's remove this func and apply the strategy I mentioned in a previous comment.